ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/SinglePeer/Indices.hs

```diff
@@ -290,20 +300,20 @@ headerPointSchedule g msgDelayInterval xs =
     -- The delay of each tipPoint is sampled from @msgDelayInterval@.
     --
     generatePerTipPointTimes
-      :: Maybe DiffTime
-      -> (DiffTime, Int)
-      -> (DiffTime, Int)
-      -> m ((DiffTime, Int), [(DiffTime, Int)])
+      :: Maybe Time
+      -> (Time, Int)
+      -> (Time, Int)
+      -> m ((Time, Int), [(Time, Int)])
     generatePerTipPointTimes mtMax (tNext0, nextHp0) (tTip, tp) = do
        t <- uniformRMDiffTime msgDelayInterval g
-       go (max tNext0 (tTip + t)) nextHp0 []
+       go (max tNext0 (addTime t tTip)) nextHp0 []
       where
-        go :: DiffTime -> Int -> [(DiffTime, Int)] -> m ((DiffTime, Int), [(DiffTime, Int)])
+        go :: Time -> Int -> [(Time, Int)] -> m ((Time, Int), [(Time, Int)])
         go tNext nextHp acc = do
           if maybe False (tNext >) mtMax || nextHp > tp then
             pure ((tNext, nextHp), reverse acc)
           else do
-            t <- (+tNext) <$> uniformRMDiffTime msgDelayInterval g
+            t <- (flip addTime tNext) <$> uniformRMDiffTime msgDelayInterval g
```

Comment:
I think the most idiomatic form is
```
(`addTime` tNext) <$> ...
```

Reply:
Thanks. Fixed in 86ddbd9b42bc3d22c2597d44855cf70411173d82.